app/lib/project/MetapolatorProject.js

```diff
@@ -588,11 +588,10 @@ define([
                                this.fontinfoFileName, this.fontinfoFile );
     };
 
-    function exportInstance(io, project, masterName, instanceName, precision) {
+    function exportInstance(io, project, instanceName, dirName, precision) {
         // returns a models/Controller
-        var model = project.open(masterName)
-          , master = model.query('master#' + masterName)
-          , dirName = instanceName
+        var model = project.open(instanceName)
+          , instance = model.query('master#' + instanceName)
```

Comment:
also this `instance` is in fact a `master`, see `'master#' + instanceName` it's contradictory.